integrations/pgvector/tests/test_document_store.py

```diff
@@ -11,10 +11,65 @@
 from haystack.document_stores.types import DuplicatePolicy
 from haystack.testing.document_store import CountDocumentsTest, DeleteDocumentsTest, WriteDocumentsTest
 from haystack.utils import Secret
+from packaging.version import parse as parse_version
 
 from haystack_integrations.document_stores.pgvector import PgvectorDocumentStore
 
 
+def get_column_type(document_store: PgvectorDocumentStore, schema_name: str, table_name: str, column_name: str) -> str:
+    """Helper function to fetch the user-defined type name of a column."""
+    sql_get_col_type = """
+        SELECT udt_name
+        FROM information_schema.columns
+        WHERE table_schema = %s
+        AND table_name = %s
+        AND column_name = %s;
+    """
+    try:
+        result = document_store._cursor.execute(sql_get_col_type, (schema_name, table_name, column_name)).fetchone()
+        return result[0] if result else ""
+    except Exception:
+        # Handle cases where the table might not exist yet or other DB errors during checks
+        return ""
+
+
+def check_pgvector_version(min_version_str="0.7.0"):
+    """
+    Connects to the test DB and checks if the installed pgvector version
+    is less than the required minimum. Returns True if the test should be skipped.
+    """
+    connection_string = "postgresql://postgres:postgres@localhost:5432/postgres"
+    min_version = parse_version(min_version_str)
+    try:
+        with psycopg.connect(connection_string, autocommit=True, connect_timeout=5) as conn:
+            with conn.cursor() as cur:
+                cur.execute("SELECT 1 FROM pg_catalog.pg_extension WHERE extname = 'vector'")
+                if cur.fetchone() is None:
+                    return True
+
+                cur.execute("SELECT extversion FROM pg_catalog.pg_extension WHERE extname = 'vector'")
+                result = cur.fetchone()
+                if result and result[0]:
+                    current_version_str = result[0]
+                    current_version = parse_version(current_version_str)
+                    should_skip = current_version < min_version
+                    if should_skip:
+                        pass
+                    return should_skip
+                else:
+                    return True
+    except psycopg.Error:
+        return True
+    except Exception:
+        return True
```

Comment:
In our CI, we use this docker image:
https://github.com/deepset-ai/haystack-core-integrations/blob/c56f7989b91cf30a78f4672dd31b3b1266d3ec38/.github/workflows/pgvector.yml#L37

Looking at [Docker Hub](https://hub.docker.com/r/pgvector/pgvector/tags), it seems that this image incorporates the latest version of `pgvector` (currently 0.8.0), so I would remove all the logic to check the version from tests.